docs/tutorials/building-your-first-bot.mdx

```diff
@@ -0,0 +1,342 @@
+your game logic before spawning, how they are destroyed, etc... For this particular tutorial, we need to define one
+method, [`SpawnBot()`](/integrating-with-unity/seating-and-spawning-bots#canbenull-public-abstract-gameobject-spawnbotbool-latejoin-botinformation-botinformation). 
+This method will define how our bot prefab is spawned into the scene.
+
+First, create a `RegressionGames/Runtime/` folder in your `Assets` folder. Then, create a file called `CharacterBotSpawnManager.cs`.
+Double click that file to open it within your editor of choice (we recommend Rider or Visual Studio). We highly recommend
+that the file is opened as part of the solution / project as a whole, so that code completion is available.
+
+Copy and paste the following code into this file. This component allows us to drag and drop a prefab for our bots and
+a spawn point directly within our editor, and informs our SDK of these via those overridden functions.
+
+See the [full reference](/integrating-with-unity/seating-and-spawning-bots) for all the ways you can modify 
+the spawning of bots. Sometimes you may need to add this
+bot to a multiplayer network protocol, configure information like the character type, or define special start logic.
+This can all be done through the interface provided by `RGBotSpawnManager`.
+
+```cs
+using UnityEngine;
+
+using RegressionGames;
+using RegressionGames.Types;
+
+public class CharacterBotSpawnManager : RGBotSpawnManager
+{
+    
+    // Expose a field in the editor for the prefab to spawn as a bot
+    [SerializeField]
+    [Tooltip("The character to spawn")]
+    private GameObject rgBotPrefab;
+
+    // Expose a field in the editor to define where the bot should spawn
+    [SerializeField]
+    [Tooltip("Spawn point for RG Bots")]
+    private Transform botSpawnPoint;
+
+    public override GameObject SpawnBot(bool lateJoin, BotInformation botInformation)
+    {
+        // Loads the bot prefab and spawns it into the scene
+        var bot = Instantiate(rgBotPrefab, Vector3.zero, Quaternion.identity);
+        bot.transform.position = botSpawnPoint.position;
+        return bot;
+    }
+    
+    void Start()
+    {
+        // Prepares the RG SDK for spawning bots
+        RGBotServerListener.GetInstance()?.StartGame();
+        RGBotServerListener.GetInstance()?.SpawnBots();
+    }
+
+    private void OnDestroy()
+    {
+        // Stops all bots and cleans up the RG SDK
+        RGBotServerListener.GetInstance()?.StopGame();
+    }
+
+}
+```
+
+Once this component is implemented, create an empty object within the scene, rename it to `RGBotSpawner` (just so we can find it later),
+and attach this `CharacterBotSpawnManager` component to that object. Position it within the scene wherever you want to spawn your bot 
+(**make sure to set the Y coordinate to 0 so your bot does not float!**), and drag `RGBotSpawner`'s own Transform into the 
```

Comment:
I think dragging and dropping either is fine! I would need to try it out... in the meantime, fixed the long sentence issue. We could probably do it the code-only way by getting the game object, but this is written this way because it lends easier to existing games where the spawn point is already provided, and they may just be following along with their own game code.